watson-language/watson-lang.lua

```diff
@@ -0,0 +1,95 @@
+--[[
+
+# [Salesfors API](https://console.bluemix.net/docs/services/language-translator/getting-started.html) connector.
+
+### Authorization
+
+Authorization use [Public]().
+
+Watson API Explorer[Language Translator] (https://watson-api-explorer.mybluemix.net/apis/language-translator-v2)
```

Comment:
Why this link in Authorization section? You may put in the top description block.